pkg/kv/kvserver/replica_gc_queue.go

```diff
@@ -323,6 +324,10 @@ func (rgcq *replicaGCQueue) process(
 		if err := repl.store.RemoveReplica(ctx, repl, nextReplicaID, RemoveOptions{
 			DestroyData: true,
 		}); err != nil {
+			// Should never get an error from RemoveReplica.
+			const format = "error during replicaGC: %v"
+			log.Errorf(ctx, format, err)
```

Comment:
`ReportOrPanic()` will do a `log.Warningf()`, so this seems superfluous (that should arguably be a `log.Errorf()` though).

Reply:
Agree, will send a sep PR